front/src/modules/rollingStock/components/RollingStockSelector/RollingStockModal.tsx

```diff
@@ -91,6 +91,7 @@ function RollingStockModal({ ref2scroll }: RollingStockModal) {
             filteredRollingStockList={filteredRollingStockList}
             isSuccess={isSuccess}
             setIsLoading={setIsLoading}
+            hasWhiteBackground={true}
```

Comment:
almost done, whiteBG has to be enabled in RollingStockEditor, and you don't have to set a value, just `hasWhiteBackground` to avoid the warning

Reply:
ok fixed to attribute without any value